dist/libnccl-net-ofi.spec

```diff
@@ -0,0 +1,83 @@
+# Whether to build with cuda support. Default: on if neuron
+%if "%{with_cuda}" == "1" && "%{with_neuron}" == "1"
+%{error:Neuron and CUDA must not be enabled together}
+%endif
+
+%if "%{with_cuda}" == "0" && "%{with_neuron}" == "0"
+%{error:One of Neuron or CUDA must be enabled}
+%endif
+
+%if "%{with_cuda}" == "1"
+%{!?target: %global target nccl}
+%endif
+%if "%{with_neuron}" == "1"
+%{!?target: %global target nccom}
+%endif
+
+%global pname_base lib%{!?with_neuron:nccl}%{?with_neuron:nccom}-net-ofi
+%global pname %{pname_base}%{?with_platform_aws:-aws}
+
+%if "%{with_platform_aws}"
+%global _prefix /opt/amazon/%{pname_base}
+%endif
+
+# (CUDA only) what toolkit package to declare a build dependency on. Default: 12-6
+%{!?_cuda_toolkit_version: %global _cuda_toolkit_version 12-6}
+
+Name:           %{pname}
+Version:        1.11.0pre
+Release:        0%{?dist}
+Summary:        NCCL + libfabric compatibility layer
+License:        Apache-2.0
+URL:            https://github.com/aws/aws-ofi-nccl
+Source0:        libnccl-net-ofi_1.11.0pre45a7e10.dirty.tar.gz
```

Comment:
really don't want to go down this path.  As I've said many times before, use Libfabric as your model here.

Reply:
This is something that is autofilled by Packit; the version that is checked in here does not reflect the actual rpmspec embedded in the SRPM. I can make that more clear by putting a placeholder value here.